python/python/tests/test_async_client.py

```diff
@@ -7303,6 +7303,365 @@ async def test_lcs_idx(self, redis_client: GlideClient):
         with pytest.raises(RequestError):
             await redis_client.lcs_idx(key1, lcs_non_string_key)
 
+    # Cluster scan tests
```

Comment:
please move all scan tests to a dedicated file- test_scan.py

Reply:
What an honor